packages/core/src/components/Table/Table.tsx

```diff
@@ -130,13 +130,14 @@ const useToolbarStyles = makeStyles<BackstageTheme>(theme => ({
 }));
 
 const convertColumns = (columns: TableColumn[]): TableColumn[] => {
-  return columns.map(column => {
+  return columns.map((column) => {
+    const theme = useTheme<BackstageTheme>();
```

Comment:
Hooks should always be called unconditionally and directly within the scope of a component function or another custom  hook. You want to call this hook in the component and then pass it in to this function instead.

Reply:
Nice catch! Gonna update it in a bit :)